main.py

```diff
@@ -11,23 +12,53 @@ def index():
     return render_template("index.html")
 
 
-@app.route("/search", methods=["GET"])
+@app.route("/search", methods=["GET", "POST"])
 def search():
-    app.logger.info(f"search(): GET {request.full_path}.")
-    keyword = request.args.get("query", None)
-    if keyword:
-        context_dict = {
-            "subtitle": f"Search Result for {keyword}",
-            "keyword": keyword,
-            "message": None
-        }
+    if request.method == "GET":
+        app.logger.info(f"search(): GET /{request.full_path}")
+    else:
+        app.logger.info(f"search(): POST /{request.full_path}")
+    context_dict = {
+        "subtitle": "a service that displays search results."
```

Comment:
subtitleって、ページのタイトルに使われる感じだから、
```suggestion
        "subtitle": "Search results for {{ keyword }}"
```
とか？

Reply:
そうですね、修正します。